### PR TITLE
[diem-framework] Send NetworkIdentity events to a central event stream

### PR DESCRIPTION
**Pull request by gregnazario**
## Motivation

Multiple event streams don't scale, so copying DiemId, we'll write to one stream.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/diem/diem/blob/main/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

Move Prover ran locally and I updated the functional tests

### CODE REVIEW

**Comment by gregnazario** [comment]
@davidiw I'm still blocked on some pieces that @tzakian is working on for making this have a smoke test.  Are there any comments you have on the code as it is right now?

File: language/diem-framework/modules/NetworkIdentity.move [hunk 1]
```diff
@@ -133,23 +163,27 @@ module DiemFramework::NetworkIdentity {
...
         let has_change = remove_members_internal(identities, &to_remove);
         if (has_change) {
-            Event::emit_event(&mut identity.identity_change_events, NetworkIdentityChangeNotification {
-                identities: *&identity.identities,
-                time_rotated_seconds: DiemTimestamp::now_seconds(),
-            });
+            Event::emit_event(
+                &mut borrow_global_mut<NetworkIdentityEventHandle>(@TreasuryCompliance).identity_change_events,
+                NetworkIdentityChangeNotification {
+                    identities: *&identity.identities,
```
Comment (davidiw) [inline language/diem-framework/modules/NetworkIdentity.move:169]: seems we are missing the signers account /  address?

Comment (gregnazario) [reply language/diem-framework/modules/NetworkIdentity.move]: Yeah, this is missing.  I've fixed it.

**Review by davidiw — APPROVED** [review]
looks good

/land

**Pull request merged**